proto/cosmos/distribution/v1beta1/tx.proto

```diff
@@ -29,7 +29,12 @@ service Msg {
 
   // FundCommunityPool defines a method to allow an account to directly
   // fund the community pool.
-  rpc FundCommunityPool(MsgFundCommunityPool) returns (MsgFundCommunityPoolResponse);
+  //
+  // Deprecated: Use x/protocolpool module's FundCommunityPool instead.
+  // Since: cosmos-sdk 0.50
```

Comment:
do we need this field? or is it deprecated since?

Reply:
it is deprecated since 0.50 (since addition of this PR)